doc/reference/power_management/index.rst

```diff
@@ -90,109 +89,97 @@ is a thread ready to run or if an external event occurred.
 System Power Management
 ***********************
 
-This consists of the hook functions that the power management subsystem calls
-when the kernel enters and exits the idle state, in other words, when the kernel
-has nothing to schedule. Enabling system power management compels Zephyr kernel
-scheduler to work in tickless idle mode (see :option:`CONFIG_TICKLESS_IDLE`).
+When the kernel enters the idle state, in other words, when the kernel has
+nothing to schedule the System Power Management, if enabled via
+:option:`CONFIG_SYS_POWER_MANAGEMENT` Kconfig option, can put the system in one
+of the supported power states based on the selected power management policy.
 
-Suspend Hook function
-=====================
+It is an application responsibility to set up a wake up event. A wake up event
```

Comment:
Could you please tell here how the app should hook to the PM in order to setup the wake-up event.

Reply:
The problem of setting a wake up event was largely ignored when designing the power management subsystem. This part we still need to discuss and agree upon. Nevertheless, as far as I can tell looking at the current source code, the paragraph holds true.